### PR TITLE
doc: more details

### LINKED ISSUE #4: Can't run composer install
Hello,

It seems that since you added the frankenphp runtime in `composer.json`, I can't run `composer install` from `README` because `composer` is using PHP 8.1 and PHP 8.2 is needed.

```
    - Root composer.json requires php >=8.2 but your php version (8.1.11) does not satisfy that requirement.
```

So I built a custom composer image :
```
FROM php:8.2.0RC4
RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
```

And now I have : 
```
  Problem 1
    - laminas/laminas-code is locked to version 4.7.0 and an update of this package was not requested.
    - laminas/laminas-code 4.7.0 requires php >=7.4, <8.2 -> your php version (8.2.0RC4) does not satisfy that requirement.
  Problem 2
    - laminas/laminas-code 4.7.0 requires php >=7.4, <8.2 -> your php version (8.2.0RC4) does not satisfy that requirement.
    - friendsofphp/proxy-manager-lts v1.0.12 requires laminas/laminas-code ~3.4.1|^4.0 -> satisfiable by laminas/laminas-code[4.7.0].
    - friendsofphp/proxy-manager-lts is locked to version v1.0.12 and an update of this package was not requested
```

Well, `laminas/laminas-code` is not yet available to PHP 8.2.

So, is it a dead end ?

Thank you

### PR DESCRIPTION
**Pull request by COil**
Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,12 +3,29 @@
 A demo app using [FrankenPHP](https://frankenphp.dev) that uses
 Symfony and API Platform.
 
+## Prerequisite
```
Comment (dunglas) [inline README.md:6]: Can we drop this section? Actually, there are no prerequisites, this project works even without FrankenPHP.

File: README.md [hunk 2]
```diff
@@ -19,4 +36,23 @@ docker run \
     dunglas/frankenphp
 ```
 
-The repository also includes [a benchmark](benchmark/) comparing FrankenPHP and PHP-FPM.
+Build the database (it uses a local SQLite database stored in `var/data.db`):
+
+```
+bin/console doctrine:database:create
```
Comment (dunglas) [inline README.md:42]: This isn't necessary with SQLite.

File: README.md [hunk 3]
```diff
@@ -19,4 +36,23 @@ docker run \
...
+bin/console doctrine:migrations:migrate --no-interaction
+```
+
+Insert some data (or use the [API Platform](https://localhost/api/monsters) swagger interface):
```
Comment (dunglas) [inline README.md:46]:
```suggestion
Insert some data (or use the [API Platform's Swagger UI](https://localhost/api/monsters)):
```

**Review by dunglas — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (COil) [reply README.md:42]: Indeed.

File: README.md (hunk 1 above)
Comment (COil) [reply README.md:6]: Yes, but the goal is to test FrankenPHP, no? And we can't without docker.

Comment (dunglas) [reply README.md:6]: Docker is easier but entirely optional: https://github.com/dunglas/frankenphp/blob/main/docs/compile.md

Comment (COil) [reply README.md]: Done.

**Review by dunglas — APPROVED** [review]

**Pull request merged**